packages/ckeditor5-ckfinder/src/ckfinderediting.js

```diff
@@ -31,7 +32,7 @@ export default class CKFinderEditing extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ Notification, ImageEditing, LinkEditing ];
+		return [ Notification, ImageEditing, ImageBlock, LinkEditing ];
```

Comment:
Why `ImageBlock` is needed here? To keep backward compatibility? But what if some integrator will want to use only `ImageInline`? Maybe this shouldn't be here.

Reply:
Hm, `ImageEditing` needs `ImageBlock` or `ImageInline` to handle images, so I've assumed that it is necessary everywhere where `ImageEditing` is used for compatibility. Do we need to document it for this and similar features or it is obvious that a user will always import Image, ImageBlock, or ImageInline along with it? (btw. tests passes without ImageBlock)

Reply:
> *   Those who want the old feature set (only block, no inline images) will need to update their configs and replace `Image` with `ImageBlock` (needs to be documented in the guide).
> *   Those who want only inline images will import `ImageInline` instead of `Image` (needs to be documented in the guide).

Reply:
Done.